score-lang-compiler/src/main/java/org/openscore/lang/compiler/modeller/transformers/ForTransformer.java

```diff
@@ -21,7 +21,10 @@
 @Component
 public class ForTransformer implements Transformer<String, ForLoopStatement>{
 
+    // case: value in variable_name
     private final static String FOR_REGEX = "^(\\s+)?(\\w+)\\s+(in)\\s+(\\w+)(\\s+)?$";
+    // case: (key value)
+    private final static String KEY_VALUE_PAIR_REGEX = "^(\\s+)?(\\w+)(, )(\\w+)(\\s+)?$";
```

Comment:
I would add optional spaces before and after the comma `^(\\s+)?(\\w+)(\\s+)?(,)(\\s+)?(\\w+)(\\s+)?$`
to match also `k  ,v` or `k,v` ..